internal/core/sapsystem/sapsystem.go

```diff
@@ -227,12 +231,28 @@ func findInstances(fs afero.Fs, sapPath string) ([][]string, error) {
 	return instances, nil
 }
 
-func getProfilePath(sysPath string) string {
-	return path.Join(sysPath, "SYS", "profile", sapDefaultProfile)
+// FindProfiles returns the latest profile file names in the /sapmnt/${SID}/folder folder
```

Comment:
Here you refer to the "latest" profile names and a bit below, on line :246 you output "New SAP profile found"... 
Are these really new or will they be always found everytime this code is run?

Reply:
Well, latest in this context means the "latest valid SAP profile".
SAP has its own backing up system, where old backup files are stored in the same folder, with `.x` terminology (where x is a digit). It is a bit different for the `DEFAULT.PFL` but the concept is the same.
The logging simply prints all found profiles.

This code is deterministic, it will always return the same, as long as the Filesystem is the same of course

Reply:
Thanks for the clarification!